model/task_lifecycle.go

```diff
@@ -73,6 +74,15 @@ func SetActiveState(t *task.Task, caller string, active bool) error {
 			})
 		}
 
+		version, err := VersionFindOneId(tasksToActivate[0].Version)
+		if err != nil || version == nil {
```

Comment:
Unfortunately you can't combine these two; if version is nil because no version could be found then err is also nil, and 79 will return a nil error